docker-compose.yml

```diff
@@ -46,19 +46,21 @@ services:
       - "2181:2181"
 
   elasticsearch:
-    image: elasticsearch:latest
+    image: elasticsearch:2.3.1
```

Comment:
I used old versions to have a local build running as closely to prod as possible. This was before I knew about docker compose overrides.